### DISCUSSION TITLE
The problem with printing on Xerox printer

### DISCUSSION THREAD

**Discussion by BIMSteam** [sumatrapdfreader/sumatrapdf#4590] [Q&A]
There are two printers installed in the system – Xerox and Canon. Printing on the Canon works fine, but the Xerox prints as if its photoconductor unit has failed – the entire page has a large number of white horizontal lines. At first, I thought it was a driver issue for the Xerox, but all other programs print on it correctly, the problem is only with Sumatra.

**Comment by GitHubRulesOK** [comment]
Its a long known driver problem seen with Xerox (search white stripes) and not other drivers and has always been an issue since SumatraPDF hands rendered images over to Windows and Windows consumes vast amounts of memory that Xerox tries to avoid overburdening the printers by optimisation! 
Basically use a different leaner optimised method like Acrobat or GhostScript that avoid Windows PAI methods.
